website/src/blogs/2017-08-31-ara-recording-ansible-playbook-runs.md

```diff
@@ -0,0 +1,44 @@
+---
+title: ARA - Recording Ansible Playbook Runs
+author: Karthik Satchitanand
+author_info: Karthik has been into the Design and Development of tools for infrastructure as code, software testing performance & benchmarking & chaos engineering.
+tags: Ansible, Automation, DevOps, Openebs
+date: 31-08-2017
+excerpt: Ansible playbooks can generate quite a lot of console data. Add the -v (I do it by default !) and you have a quite a lot to scroll through.
+---
+
+Ansible playbooks can generate quite a lot of console data. Add the -v (I do it by default !) and you have quite a lot to scroll through. Oftentimes, one tends to feel the need for a better reporting mechanism — one which is easy to scour for specific task status, whilst having the luxury to extract additional debug info for the same if needed. Also, it would be great if this aid is available for playbook runs across time, i.e., for older playbook runs as well. This would be especially beneficial when running ansible-based CI suites, like @OpenEBS, where the application deployments, test setup & execution are driven by Ansible.
```

Comment:
```suggestion
Ansible playbooks can generate quite a lot of console data. Add the -v (I do it by default!) and you have quite a lot to scroll through. Oftentimes, one tends to feel the need for a better reporting mechanism — one which is easy to scour for specific task status, whilst having the luxury to extract additional debug info for the same if needed. Also, it would be great if this aid is available for playbook runs across time, i.e., for older playbook runs as well. This would be especially beneficial when running ansible-based CI suites, like @OpenEBS, where the application deployments, test setup & execution are driven by Ansible.
```